codeship-steps.yml

```diff
@@ -3,16 +3,16 @@
   - type: parallel
     steps:
     - name: rspec
-      service: web
+      service: web_codeship_example
```

Comment:
I think we can use `web_codeship_example_{lang}` here - which would allow us to fuzzy match `web_codeship_example*` and then split into languages as well - what do you think?